app/javascript/components/admin/server_recordings/ServerRecordings.jsx

```diff
@@ -51,15 +51,18 @@ export default function ServerRecordings() {
                           isLoading={isLoading}
                           RecordingRow={ServerRecordingRow}
                         />
+                        {!isLoading
+                          && (
+                            <div className="table-pagination-wrapper">
+                              <Pagination
+                                page={serverRecordings.meta.page}
+                                totalPages={serverRecordings.meta.pages}
+                                setPage={setPage}
+                                borders
```

Comment:
With the `table-pagination-wrapper` css class in place I believe that the borders property is absurd,